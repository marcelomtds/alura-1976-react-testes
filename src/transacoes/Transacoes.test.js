import { render } from "@testing-library/react";
import React from "react";
import Transacoes from "./Transacoes";

describe("Component de transações do extrato", () => {
  it("O snapshot do component deve permanecer sempre o mesmo", () => {
    const { container } = render(
      <Transacoes
        transacoes={[{ data: "10/10/2022", tipo: "deposito", valor: "100.00" }]}
      />
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
