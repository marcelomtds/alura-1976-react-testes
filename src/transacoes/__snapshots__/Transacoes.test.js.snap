// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component de transações do extrato O snapshot do component deve permanecer sempre o mesmo 1`] = `
<div
  data-testid="transacoes"
>
  <div
    class="transacao-container"
  >
    <p>
      10/10/2022
    </p>
    <p />
    <p>
      R$ 
      100.00
    </p>
  </div>
</div>
`;
